.circleci/copy-configs

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -x
+set -e
+
+# In your ~/.aws/config and ~/.aws/credentials, we expect this profile
+# It must have S3 read-only access to the config bucket.
+readonly cli_profile='amplify_sdk_test'
+
+# This bucket contains a collection of config files that are used by the
+# integration tests. They contain sensitive
+# tokens/credentials/identifiers, so are not published publicly.
+readonly config_bucket='amplify-circleci-android-payload'
+
+# Mac OS X's antiquated version of bash doesn't have associative arrays.
+# So, use two simple arrays and join on index.
+# sources are remote URLs, targets are local file paths
+readonly sources=(
+    "s3://$config_bucket/awsconfiguration.json"
+    "s3://$config_bucket/amplifyconfiguration-analytics.json"
+    "s3://$config_bucket/amplifyconfiguration-core.json"
+    "s3://$config_bucket/amplifyconfiguration-api.json"
+    "s3://$config_bucket/amplifyconfiguration-datastore.json"
```

Comment:
any ideas around process for when API key expires?

Reply:
I don't have any immediate ideas for that, no. So long as we use the maximum expiration time on these test keys, though, it should be pretty low-effort to have on-call rotate them once per year. Since they aren't used for production, I think it should be acceptable (not desirable) for our test APIs to experience some occasional downtime.